<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vieBBArmourSourceWidget</class>
 <widget class="QWidget" name="vieBBArmourSourceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vieBBArmourSourceWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Physical</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Power</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Magic</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Evil</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Undead</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Dexterity</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="spinBox_2"/>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBox_3"/>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinBox_4"/>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="spinBox_5"/>
   </item>
   <item row="1" column="6">
    <widget class="QSpinBox" name="spinBox_6"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionNameBoxChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionPhysicalBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionPowerBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionMagicBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionEvilBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionUndeadBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>vieBBArmourSourceWidget</receiver>
   <slot>actionDexterityBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>554</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionNameBoxChanged(QString)</slot>
  <slot>actionPhysicalBoxChanged(int)</slot>
  <slot>actionPowerBoxChanged(int)</slot>
  <slot>actionMagicBoxChanged(int)</slot>
  <slot>actionEvilBoxChanged(int)</slot>
  <slot>actionUndeadBoxChanged(int)</slot>
  <slot>actionDexterityBoxChanged(int)</slot>
 </slots>
</ui>
